terraform-plans/files/github/snap_promote.yaml

```diff
@@ -1,14 +1,22 @@
-name: Promote snap to other tracks and channels
+name: Promote snap to default track, standard risk levels.
 
 on:
   workflow_dispatch:
     inputs:
       destination-channel:
         description: 'Destination Channel, e.g. latest/candidate'
         required: true
+        type: choice
+        options:
+          - 'latest/candidate'
+          - 'latest/stable'
```

Comment:
Can we use same option as we did for charm. Thanks

Reply:
It's updated.